syncer/samples/multi-servicecenters/multi-servicecenters-ZH.md

```diff
@@ -0,0 +1,251 @@
+异构服务中心
+-------  
+[Introduction to English](./multi-servicecenters.md)  
+
+本案例模拟了异构服务中心之间的通讯场景，包括以下四个部分：  
+- EurekaServer： Eureka服务中心  
+- AccountServer：注册到Eureka的账号服务  
+- Servicecomb-ServiceCenter：Servicecomb-ServiceCenter服务中心  
+- HelloServer：注册到Servicecomb-ServiceCenter，需要使用AccountServer的服务  
+
+在传统的环境中，不同服务中心之间的实例是不能互相发现并访问的。当我们使用Syncer后，这将很容易实现。如下图：  
+![image](../../images/MultiServicecenters.png?raw=true)   
```

Comment:
Update the Eureka spelling in the image

Reply:
@asifdxtreme I was fixed it. Pls review. Thx!